CorsixTH/Lua/world.lua

```diff
@@ -130,6 +130,7 @@ function World:World(app)
   self.floating_dollars = {}
   self.game_log = {} -- saves list of useful debugging information
   self.savegame_version = app.savegame_version
```

Comment:
Please write for both variables that you use their meaning here in a line of text. In that way you avoid that you or someone else has to reverse engineer the meaning that you already found or created.